api/camunda/camunda-openapi.yaml

```diff
@@ -618,6 +592,41 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /tenants/{tenantId}/users/search:
+    post:
+      tags:
+        - Tenant
+        - User
+      operationId: searchUsersForTenant
+      summary: Query users for tenant
+      description: Retrieves a filtered and sorted list of users for a specified tenant.
+      parameters:
+        - name: tenantId
+          in: path
+          required: true
+          description: The unique identifier of the tenant.
+          schema:
+            type: string
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UserSearchQueryRequest"
+      responses:
+        "200":
+          description: The search result of users for the tenant.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UserSearchResult"
+            application/vnd.camunda.api.keys.number+json:
+              schema:
+                $ref: "#/components/schemas/UserSearchResult"
+            application/vnd.camunda.api.keys.string+json:
+              schema:
+                $ref: "#/components/schemas/UserSearchResult"
```

Comment:
There are multiple issues with this endpoint. I have [pointed them out in the causing PR](https://github.com/camunda/camunda/pull/27382#pullrequestreview-2625769229) and have asked the devs for a follow-up. I have also [informed the respective team](https://camunda.slack.com/archives/C06UYJMMETZ/p1739948563334569) in their Slack channel.
@pepopowitz, I think we need to look into automatically assigning the API reviewers to PRs that adjust the OpenAPI spec.